src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -52,9 +52,14 @@ let NodeColorFillHoverDegraded: PFColorVal;
 let NodeColorFillHoverFailure: PFColorVal;
 const NodeHeight = '25px';
 const NodeIconCB = icons.istio.circuitBreaker.className; // bolt
+const NodeIconFaultInjection = icons.istio.faultInjection.className; // ban
 const NodeIconMS = icons.istio.missingSidecar.className; // exclamation
 const NodeIconRoot = icons.istio.root.className; // alt-arrow-circle-right
 const NodeIconVS = icons.istio.virtualService.className; // code-branch
+const NodeIconRequestRouting = icons.istio.requestRouting.className; // code-branch
+const NodeIconRequestTimeout = icons.istio.requestTimeout.className; // clock
+const NodeIconTCPTrafficShifting = icons.istio.tcpTrafficShifting.className; // share-alt
```

Comment:
Until we are going to use distinctive icons I don't think we need this duplicate definition.  Let's just have NodeIconTrafficShifting and we can remove icons.istio.tcpTrafficShifting.